Makefile

```diff
@@ -103,7 +103,7 @@ podman-dev-up:
 		--env LOG_LEVEL="debug"\
 		--expose 8080\
 		--volume $$(git rev-parse --show-toplevel)/:/src/claircore/:z\
-		quay.io/claircore/golang:1.13.3\
+		quay.io/claircore/golang:$(GO_VERSION)\
```

Comment:
should a default at top of file be declared?

Reply:
It's defined just below this, above the `baseimage` target